src/Gui/NaviCube.cpp

```diff
@@ -269,6 +275,8 @@ NaviCubeImplementation::NaviCubeImplementation(
 	OnChange(*hGrp, "HiliteColor");
 	OnChange(*hGrp, "ButtonColor");
 	OnChange(*hGrp, "CubeSize");
+	OnChange(*hGrp, "BorderWidth");
+	OnChange(*hGrp, "BorderColor");
```

Comment:
Why is the BorderColor important to fix the rendering? The PR fixes the issue also without a border color.

I committed this now nevertheless but please let us only change things that have to be changed to fix an issue.